cf-oscore/src/main/java/org/eclipse/californium/oscore/OSCoreCtx.java

```diff
@@ -27,6 +27,7 @@
 
 import javax.crypto.Mac;
 import javax.crypto.spec.SecretKeySpec;
+import javax.xml.bind.DatatypeConverter;
```

Comment:
[requires "--add-modules java.xml.bind" for newer jvm](https://github.com/http-kit/http-kit/issues/356)

Please consider to replace that with 
[StringUtil](https://github.com/eclipse/californium/blob/master/element-connector/src/main/java/org/eclipse/californium/elements/util/StringUtil.java)

Reply:
Yes, that worked just the same. Fixed in commit 81c1fdc07c6c06cf58e8bddb3e05be99029e1bf7